lib/profileAgents.js

```diff
@@ -602,6 +609,104 @@ async function _updateMeterController({meterId, controller}) {
   }
 }
 
+async function _getProfileAgentRecords({accountId, includeSecrets}) {
+  const query = {'profileAgent.account': accountId};
+  const projection = {_id: 0};
+  const collection = getCollection('profile-profileAgent');
+  let records = await collection.find(query, {projection}).toArray();
+  const zcapRefreshed = await _refreshProfileAgentZcaps({
+    profileAgentRecords: records
+  });
+  if(zcapRefreshed) {
+    records = await collection.find(query, {projection}).toArray();
+  }
+  if(!includeSecrets) {
+    records = await removeSecretsFromRecords({records});
+  }
+  return records;
+}
+
+async function _refreshProfileAgentZcaps({profileAgentRecords}) {
+  let zcapRefreshed = false;
+  const records = klona(profileAgentRecords);
+  const now = Date.now();
+  for(const record of records) {
+    if(!record.profileAgent.zcaps) {
+      continue;
+    }
+    const {zcaps} = record.profileAgent;
+    for(const zcapName in zcaps) {
+      if(zcapName !== 'profileCapabilityInvocationKey') {
```

Comment:
```suggestion
      if(zcapName === 'profileCapabilityInvocationKey') {
        continue;
```

This prevents additional idents that we don't need in the code.

Reply:
Addressed [21b16f5](https://github.com/digitalbazaar/bedrock-profile/pull/99/commits/21b16f55ca815b5286069f82e668f36b6064bbfa)